netbox/templates/dcim/interface.html

```diff
@@ -355,6 +359,11 @@ <h2 class="card-header">
       {% include 'inc/panel_table.html' with table=vlan_table heading="VLANs" %}
     </div>
   </div>
+  <div class="row mb-3">
+    <div class="col col-md-12">
+      {% include 'inc/panel_table.html' with table=vlan_translation_table heading="VLAN Translation" %}
+    </div>
+  </div>
```

Comment:
We probably want to include this only if a translation policy has been defined for the interface.